build.gradle

```diff
@@ -1,19 +1,36 @@
-apply plugin: 'com.android.library'
-
-group = 'org.slamon.android'
```

Comment:
actually, could the group be just *org.slamon*, as it's in the java lib? or would it conflict somehow?